dev/tests/functional/standalone_bootstrap.php

```diff
@@ -74,3 +74,9 @@
 if (!(bool)$debug_mode && extension_loaded('xdebug')) {
     xdebug_disable();
 }
+
+// include suite php files
+$groupPath = PROJECT_ROOT . '/src/Magento/FunctionalTestingFramework/Group';
+foreach (glob($groupPath. '/*.php') as $suitePhpFile) {
+    include_once $suitePhpFile;
+}
```

Comment:
The `Group` is in MFTF namespace. I am not sure why we need to require it in specifically. Do you see the tests file being generated for the suite?